README.md

```diff
@@ -18,10 +18,11 @@ npm i @bayer/ol-kit ol@4.6.5 react react-dom styled-components --save
 It's easy to start building map apps with ol-kit. For simple projects the following will get you started:
 ```javascript
 import React from 'react'
-import { Map, Popup, Controls, zoomToExtent } from '@bayer/ol-kit'
+import { Map, Popup, Controls } from '@bayer/ol-kit'
+import { centerAndZoom } from '@bayer/ol-kit/core/Map/utils'
```

Comment:
can this not be exported straight off the root of `@bayer/ol-kit`? if not, it should

Reply:
nice you discovered a bug- this export needs to have `centerAndZoom` added to it (preferably alphabetical order) wanna add that to your PR to fix the export? https://github.com/MonsantoCo/ol-kit/blob/11d302b47bb04cf66b3761993cded7b921a2d82c/src/index.js#L2

Reply:
Can we test the export signature to ensure its what we expect?

Reply:
@glenselle that's a good idea- @AtherBilal & I discussed a fix that will make sure this doesn't happen anymore...should be added to this PR shortly